installer/dev/MachineTypeAttributes.h

```diff
@@ -24,20 +24,22 @@ namespace MachineTypeAttributes
             auto getMachineTypeAttributes{ GetProcAddressByFunctionDeclaration(kernelbaseDll.get(), GetMachineTypeAttributes) };
             if (!getMachineTypeAttributes)
             {
-                // For now, GetMachineTypeAttributes API is available only on Windows 11 (i.e. builds 22000+).
-                THROW_LAST_ERROR();
+                DWORD getProcAddressError = GetLastError();
+                if (getProcAddressError != ERROR_PROC_NOT_FOUND)
+                {
+                    THROW_HR(HRESULT_FROM_WIN32(getProcAddressError));
+                }
+                // If execution comes here, it means the current system is not running Windows 11.
```

Comment:
Refactor similar to above suggestion (followed by "now that we have procaddress call it...")